seeq-link-sdk-debugging-agent/src/main/java/com/seeq/link/sdk/debugging/Main.java

```diff
@@ -51,15 +51,15 @@ public static void main(String[] args) {
         }
 
         Path connectorSdkRoot = executingAssemblyLocation.getParent().getParent().getParent().getParent().getParent();
-        String searchPath = connectorSdkRoot.toString() + "/*connector*/build/install/*connector*/*.jar";
+        String searchPath = connectorSdkRoot.toString() + "/*connector*/build/libs/*connector*.jar";
 
         config.setConnectorSearchPaths(searchPath);
 
         new Program().run(new com.seeq.link.agent.ClassFactory(), new ClassFactory(), config);
     }
 
     private static Path getSeeqDataFolder(){
-        String path = "src/main/resources/data";
+        String path = "seeq-link-sdk-debugging-agent/src/main/resources/data";
```

Comment:
Why did this get put here? It causes a new data folder to be used rather than the standard resources dir. It took me quite a bit to realize that a new directory was created under the module 
![image](https://github.com/seeq12/seeq-connector-sdk-java/assets/5923087/b997f08a-3801-4bf9-9753-f4d11682722f)

Reply:
That is interesting. The result now is that cloning the repo, following the directions results in a new directory being created without an agent.key file and no way for the agent to authenticate to Seeq.

Reply:
This should resolve the difference in behavior:
`    private static Path getSeeqDataFolder(){
        URL resource = Main.class.getClassLoader().getResource("data/");
        assert resource != null;
        File file = new File(resource.getPath());
        return Path.of(file.getAbsolutePath());
    }`